lib/api/settings.rb

```diff
@@ -1,3 +1,5 @@
+require "config"
```

Comment:
@abellotti I'm sneaking this in here (in a separate commit) because it should have been in my first PR